src/registrar/management/commands/load_transition_domain.py

```diff
@@ -0,0 +1,525 @@
+                    if new_entry_domain_name not in duplicate_domains:
+                        duplicate_domains.append(new_entry_domain_name)
+                if existing_domain_user_pair is not None:
+                    # DEBUG:
+                    self.print_debug(
+                        debug_on,
+                        f"""{termColors.YELLOW} DUPLICATE Verisign entries found for domain - user {termColors.BackgroundLightYellow} PAIR {termColors.ENDC}{termColors.YELLOW}:  
+                        {new_entry_domain_name} - {new_entry_email} {termColors.ENDC}"""  # noqa
+                        )
+                    if existing_domain_user_pair not in duplicate_domain_user_combos:
+                        duplicate_domain_user_combos.append(existing_domain_user_pair)
+                else:
+                    try:
+                        entry_exists = TransitionDomain.objects.exists(
+                            username=new_entry_email, domain_name=new_entry_domain_name
+                        )
+                        if(entry_exists):
+                            existing_entry = TransitionDomain.objects.get(
+                                username=new_entry_email, domain_name=new_entry_domain_name
+                            )
+
+                            if existing_entry.status != new_entry_status:
+                                # DEBUG:
+                                self.print_debug(
+                                    debug_on,
+                                    f"{termColors.OKCYAN}"
+                                    f"Updating entry: {existing_entry}"
+                                    f"Status: {existing_entry.status} > {new_entry_status}"  # noqa
+                                    f"Email Sent: {existing_entry.email_sent} > {new_entry_emailSent}"  # noqa
+                                    f"{termColors.ENDC}"
+                                    )
+                                existing_entry.status = new_entry_status
+
+                        existing_entry.email_sent = new_entry_emailSent
+                        existing_entry.save()
+                    except TransitionDomain.DoesNotExist:
+                        # no matching entry, make one
+                        new_entry = TransitionDomain(
+                            username=new_entry_email,
+                            domain_name=new_entry_domain_name,
+                            status=new_entry_status,
+                            email_sent=new_entry_emailSent,
+                        )
+                        to_create.append(new_entry)
+                        total_new_entries += 1
+
+                        # DEBUG:
+                        self.print_debug(
+                            debug_on,
+                            f"{termColors.OKCYAN} Adding entry {total_new_entries}: {new_entry} {termColors.ENDC}"  # noqa
+                        )
+                    except TransitionDomain.MultipleObjectsReturned:
```

Comment:
Change this except to be the except for a new try around where you do .get() in the if(entry_exists): block